src/widgets/ActivityDetails/model/mappers.ts

```diff
@@ -115,3 +130,27 @@ function convertToNumberSelectAnswer(item: SelectorItem) {
     itemId: item.id,
   }
 }
+
+function convertToMessageAnswer(item: MessageItem) {
+  return {
+    answer: null,
+    itemId: item.id,
+  }
+}
+
+function convertToDateAnswer(item: DateItem) {
+  if (!item.answer[0]) {
+    return {
+      answer: null,
+      itemId: item.id,
+    }
+  }
+
+  return {
+    answer: {
+      value: item.answer[0],
```

Comment:
Please keep in mind that the expected format for the date answer dto is 
{
  day: string;
  month: string;
  year: string;
}

Reply:
Also all the "convert" functions should have a return type, e.g. AnswerDto

Reply:
Done, take a look at DTO from the mobile app repo and change the date answer DTO to 
{
day: number
month: number
year: number
}